lib/oxidized/nodes.rb

```diff
@@ -58,7 +58,8 @@ def show node
     def fetch node, group
       with_lock do
         i = find_node_index node
-        output = self[i].output.new
+        # Use first output type
+        output = self[i].output[0].new
```

Comment:
hum, now I _really_ read the other issue and everything made sense, hahahaha!

idk, I didn't like the string splitting when YAML already have arrays:

``` yml
output:
  defaults:
    - git
    - file
```

If I configure the `file` first, then what happens with the _versions_ button in the `oxidized-web` ? :thought_balloon: